src/main/java/mate/academy/rickandmorty/init/InitDataBase.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.rickandmorty.init;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterResponseDto;
+import mate.academy.rickandmorty.exception.DataProcessException;
+import mate.academy.rickandmorty.mapper.DtoMapper;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class InitDataBase implements CommandLineRunner {
```

Comment:
your name sounds more like method name rather class name
by the way, where's your Application class?
```suggestion
public class DataInitializer implements CommandLineRunner {
```

Reply:
Here it is
<img width="1064" alt="Screenshot 2023-11-07 at 8 02 05 AM" src="https://github.com/mate-academy/jv-rick-and-morty/assets/115042054/30a67476-14c7-4320-9b99-cc0b6e205307">